public/css/style.css

```diff
@@ -537,12 +546,32 @@ footer #github-logo {
 .pager svg {
   fill: #1976D2;
 }
+
+.previous, .next {
+  transition: opacity 0.2s ease-in-out;
+}
+
 .previous {
   float: left;
 }
+
 .next {
   float: right;
 }
+
+/* Controls Offline and SignedIn Behavirous */
+body[offline] .gulliver-online-aware:not([cached]),
+body:not([signedIn]) .signin-online-aware-button {
```

Comment:
Done. Will also rename `.gulliver-online-aware` on another PR, as it touches the code in multiple places.